### PR TITLE
raidboss: Initial E9S Triggers

### PR DESCRIPTION
**Pull request by panicstevenson**
Some improvements can be made and some TODOs remain to be done, but here's some initial triggers.

### CODE REVIEW

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 1]
```diff
@@ -1,8 +1,384 @@
...
+    },
+    {
+      id: 'E9S Curse Of Darkness',
+      netRegex: NetRegexes.gainsEffect({ effectId: '953' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_, matches) => matches.duration - 3,
+      alertText: (data, matches, output) => output.text({ player: data.ShortName(matches.target) }),
+      outputStrings: {
+        text: {
+          en: 'Point Face Laser Out',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:301]:
```suggestion
          en: 'Face Laser Out',
```

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 2]
```diff
@@ -1,8 +1,384 @@
...
+      delaySeconds: 2,
+      durationSeconds: 5,
+      response: Responses.getIn(),
+    },
+    {
+      id: 'E9S Deluge Of Darkness',
+      netRegex: NetRegexes.startsUsing({ id: '55F1', source: 'Cloud Of Darkness', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 5,
+      response: Responses.aoe('alert'),
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:337]:
```suggestion
      response: Responses.bigAoe('alert'),
```

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 3]
```diff
@@ -1,8 +1,384 @@
...
+      },
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Sides',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 11,
+      response: Responses.goSides(),
+    },
+    {
+      id: 'E9S The Second Art Of Darkness Right -> Stacks',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:204]: Where is left -> protean and right -> protean?

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 4]
```diff
@@ -1,8 +1,384 @@
...
+      durationSeconds: 18,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Left -> Protean -> Right',
+        },
+      },
+    },
+    {
+      // TODO: Unconfirmed
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:354]: I am skeptical of this.  Can you get more data or remove for now? Mostly I don't want to give anybody an incorrect left/right call.

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 5]
```diff
@@ -1,8 +1,384 @@
...
+    },
+    {
+      id: 'E9S Anti-Air Phaser Unlimited Out',
+      netRegex: NetRegexes.startsUsing({ id: '561[23]', source: 'Cloud Of Darkness', capture: false }),
+      delaySeconds: 1,
+      durationSeconds: 6,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Go Tethered Wall Then Out',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:162]: What about `Out -> Stacks -> Sides` and `Out -> Tank Spread -> Sides`, depending on roll?

File: ui/raidboss/data/05-shb/raid/e9s.js [hunk 6]
```diff
@@ -1,8 +1,384 @@
...
+        if (data.role === 'tank')
+          return output.stayOut();
+      },
+      alertText: function(data, _, output) {
+        if (data.role !== 'tank')
+          return output.healerStacks();
+      },
+      outputStrings: {
+        stayOut: {
+          en: 'Stay Out',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e9s.js:180]: maybe "Tank Spread"?

File: ui/raidboss/data/05-shb/raid/e9s.js (hunk 3 above)
Comment (panicstevenson) [reply ui/raidboss/data/05-shb/raid/e9s.js]: Removed for now

**Pull request merged**